src/index.js

```diff
@@ -129,16 +130,19 @@ const idyll = (inputPath, opts, cb) => {
       const children = node[2] || [];
       if (ignoreNames.indexOf(name) === -1 && checkedComponents.indexOf(name) === -1) {
         if (inputConfig.components[name]) {
-          outputComponents.push(`"${name}": require('${path.resolve(path.join(inputDirectory, inputConfig.components[name])).replace(/\\/g, '\\\\')}')`);
+          const compPath = path.parse(path.join(inputDirectory, inputConfig.components[name]));
+          outputComponents.push(`'${name}': require('${path.join(path.relative(TMP_PATH, compPath.dir), compPath.base)}')`);
         } else if (customComponents.indexOf(name + '.js') > -1) {
-          outputComponents.push(`"${name}": require('${path.join(CUSTOM_COMPONENTS_FOLDER, name).replace(/\\/g, '\\\\')}')`);
+          outputComponents.push(`'${name}': require('${path.relative(TMP_PATH, path.join(CUSTOM_COMPONENTS_FOLDER, name))}')`);
         } else if (components.indexOf(name + '.js') > -1) {
-          outputComponents.push(`"${name}": require('${path.join(DEFAULT_COMPONENTS_FOLDER, name).replace(/\\/g, '\\\\')}')`);
+          outputComponents.push(`'${name}': require('${path.relative(TMP_PATH, path.join(DEFAULT_COMPONENTS_FOLDER, name))}')`);
```

Comment:
We need to keep these `.replace(/\\/g, '\\\\')` to make the tests pass on windows